modules/ossm-configuring-the-threescale-wasm-auth-module.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-configuring-the-threescale-wasm-auth-module_{context}"]
+= Configuring the `threescale-wasm-auth` module
+
+NOTE: To configure the 3scale `threescale-wasm-auth` module, refer to {ProductName} custom resources for instructions on adding adapter parameters to the custom resource file.
+
+.The Service Mesh extension
+
+{ProductShortName} provides a link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources[custom resource definition] to specify and apply _Proxy-WASM_ extensions to sidecar proxies. This custom resource is referred to as link:https://docs.openshift.com/container-platform/4.8/service_mesh/v2x/ossm-extensions.html[`ServiceMeshExtension`]. This custom resource will be applied to a set of workloads that require HTTP API management with 3scale.
+
+.Prerequisites
+
+* Identify a kubernetes workload and namespace on your {ProductShortName} deployment that you will apply this module.
+* You must have a 3scale account (link:https://www.3scale.net/signup[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.11/html-single/installing_3scale/index#install-threescale-on-openshift-guide[3scale 2.11 On-Premises]) with a matching service and relevant applications and metrics defined.
+* If you apply the module to the `productpage` microservice in the `bookinfo` namespace, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/service_mesh/index#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo sample application]. The following example is the format for the custom resource for `threescale-wasm-auth`:
++
+----
+apiVersion: maistra.io/v1alpha1
+kind: ServiceMeshExtension
+metadata:
+  name: threescale-wasm-auth
+  namespace: bookinfo
+spec:
+  workloadSelector:
+    labels:
+      app: productpage
+  config: <yaml_configuration>
+  image: quay.io/3scale/threescale-wasm-auth:1.0
```

Comment:
This image is definitely wrong as I don't believe we will use quay for released image

Reply:
@pehala Thanks for catching this.

@rahulanand16nov Let me know where I can get the correct image path when it's avialable. Cheers.

Reply:
@dfennessy It should be `registry.redhat.io/openshift-service-mesh/3scale-auth-wasm-rhel8`